llvm/lib/TargetParser/RISCVISAInfo.cpp

```diff
@@ -717,6 +716,15 @@ RISCVISAInfo::parseArchString(StringRef Arch, bool EnableExperimentalExtension,
     } while (!Ext.empty());
   }
 
+  // We add Zicsr/Zifenci as final to allow duplicated "zicsr"/"zifencei".
```

Comment:
Nit, give an example that it's duplicated with g?

```suggestion
  // We add Zicsr/Zifenci as final to allow duplicated "rv64g_zicsr_zifencei".
```